app/constants.js

```diff
@@ -0,0 +1,3 @@
+exports.PREVIOUS_MESSAGE = 'Making request';
+exports.MESSAGE_OK = 'Request made correctly, sending...';
+exports.MESSAGE_API_FAILED = 'Failed to consume the album API';
```

Comment:
`MESSAGE_API_ALBUM_FAILED`

Reply:
Always we try to use descriptive names to be clearer to other programmers who see our code